packages/dev/docs/pages/releases/2022-05-27.mdx

```diff
@@ -20,15 +20,32 @@ date: 2022-05-27
 Ever since React 18 was announced last July, compatibility has been front of mind for the team. We are extremely pleased to announce that our library now officially supports React 18! Special thanks to [@eps1lon](https://github.com/eps1lon) for their
 help with getting our tests working with React 18.
 
-A major version bump to our workflow icons is included in this release, so be sure to upgrade accordingly. Other updates include added support for emphasized Tabs and various bug fixes to our existing components and hooks. Thank you to all who contributed to this release!
+A major version bump to our workflow icons is included in this release, so be sure to upgrade accordingly. The changes in this version update are as follows:
```

Comment:
I feel like this should be a section down below instead of being in the overview. Overview should say something like
```
A major version bump to our workflow icons is included in this release because our icons team released a major version, so be sure to upgrade according to the "Name of section" below.
```